cmd/release-controller/types.go

```diff
@@ -360,6 +367,13 @@ const (
 	// releaseAnnotationFromImageStream specifies the imagestream
 	// a release was promoted from. It has the format <namespace>/<imagestream name>
 	releaseAnnotationFromImageStream = "release.openshift.io/from-image-stream"
+
+	// releaseAnnotationBugsVerified indicates whether or not the release has been
+	// processed by the BugzillaVerifier
+	releaseAnnotationBugsVerified = "release.openshift.io/bugs-verified"
+	// publishTypeBugzilla is the key for the publish step to verify bugzilla bugs
+	// in accepted releases
+	publishTypeBugzilla = "verifyBugs"
```

Comment:
Why is this a constant?  Don't create global constants that are only used in one spot.